projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.html

```diff
@@ -0,0 +1,10 @@
+<!--Scroll to top-->
+<div
+  class="cx-main"
+  [ngClass]="{ 'cx-show-scrollTop': windowScrolled }"
+>
+  <button (click)="scrollToTop()" class="cx-button-scroll-to-top">
+    <div class="cx-img-scroll-to-top"></div>
```

Comment:
No need to have 2 separate `<div>`'s for this to work. We can do it with one element and use CSS's media queries